decentraland/explorer
# chore: new profile hud style

This branch contains the re-design of Profile HUD.

WHAT IS NEW AND HOW TO TEST

[Sign in as Guest]

- You should be able to see the Claim name button(click on it to  claim your name), and next to it your alias (click on it to edit it). Both have feedback on mouse hover.
- When editing your name you should be able to see characters remaining counter.
- Sign In button is visible. Click on it to go to wallets selection modal.
- Terms and conditions should open a new taband load https://decentraland.org/terms page. Privacy policy  https://decentraland.org/privacy

[Sign in with a wallet account]

- If you don't have a claimed name you should be able to see the Claim name button(click on it to  claim your name), and next to it your alias (click on it to edit it). Both have feedback on mouse hover.
- If you have a claimed name, the claim name button and the code should disappear. It won't be posible to edit the name anymore.
- Wallet address and mana balance are now visible. Click your address to copy. After clicking it a warning message should appear.
- Hover Mana container will display a little tooltip explaining what is mana plus a learn more button.
- + button is to buy more mana.
- Terms and conditions should open a new taband load https://decentraland.org/terms page. Privacy policy  https://decentraland.org/privacy
- Log out button should load the landing.
